### DISCUSSION TITLE
Billing Bug. 13 Million Dollar Estimated Monthly Bill?

### DISCUSSION THREAD

**Discussion by aaronware** [community/community#22001] [New to GitHub]
<p>Anyone having issues with billing? I literally saw this come up for us.</p>
<p>We usually use less than 3k minutes for deploys per month</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/9/d9bb9b07c13b260f89f1fa1bd95ddeb648246a9f.png" data-download-href="/uploads/short-url/v49ymmkNQn68J5ktUGb6m2dKmnl.png?dl=1" title="Minutes Report Bug" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/3390827/181068174-2ee56bd8-fb04-4faf-ae9c-3283fcf3eea0.png" alt="Minutes Report Bug" data-base62-sha1="v49ymmkNQn68J5ktUGb6m2dKmnl" width="690" height="180" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/9/d9bb9b07c13b260f89f1fa1bd95ddeb648246a9f_2_690x180.png, https://aws1.discourse-cdn.com/github/original/3X/d/9/d9bb9b07c13b260f89f1fa1bd95ddeb648246a9f.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/9/d9bb9b07c13b260f89f1fa1bd95ddeb648246a9f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/9/d9bb9b07c13b260f89f1fa1bd95ddeb648246a9f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Minutes Report Bug</span><span class="informations">900×235 15.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p><img src="https://user-images.githubusercontent.com/3390827/181068175-7fb72d6c-928e-4fbf-9b2f-226d376e219f.png" alt="Cloudup" width="512" height="134" class="onebox"></p>

**Comment by aaronware** [comment]
<p>Also, we can’t actually do any deployments at the moment due to the billing restriction. Luckily we had  cap on spending. Hope no one has any active charges.</p>

**Comment by M1kep** [comment]
<p>Seeing the same issue here. Maybe it’s the cause of the current status for GHA.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://www.githubstatus.com/incidents/vxvyrmy9w1vp?utm_ts=1649882144">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2661819/181068176-4bcc88b2-aada-4355-bea4-e943663c5d8e.png" class="site-icon" width="32" height="32">

      <a href="https://www.githubstatus.com/incidents/vxvyrmy9w1vp?utm_ts=1649882144" target="_blank" rel="noopener nofollow ugc">githubstatus.com</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/2661819/181068177-32eb88ff-3373-4a26-a4e5-f3c51d28a231.png" class="thumbnail onebox-avatar" width="120" height="120">

<h3><a href="https://www.githubstatus.com/incidents/vxvyrmy9w1vp?utm_ts=1649882144" target="_blank" rel="noopener nofollow ugc">Incident with GitHub Actions and Codespaces</a></h3>

  <p>GitHub's Status Page - Incident with GitHub Actions and Codespaces.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by aaronware** [comment]
<p>Thanks, <a class="mention" href="/m1kep">@M1kep</a>! No reported bug when I reported it to GH SUpport and here. Glad they’re on it.</p>

**Comment by jfullstackdev** [comment]
<p>is it good now ?</p>
<p>hopefully it is</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/b/2/b265113068cd7b0589fe0b16615c08d09c3d80f5.png" data-download-href="/uploads/short-url/ps9wytRewI92y235aqBOY0SuZDv.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181068179-ce32772a-eb11-488a-a634-12abfe0a1bed.png" alt="image" data-base62-sha1="ps9wytRewI92y235aqBOY0SuZDv" width="606" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/b/2/b265113068cd7b0589fe0b16615c08d09c3d80f5_2_606x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/b/2/b265113068cd7b0589fe0b16615c08d09c3d80f5_2_909x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/b/2/b265113068cd7b0589fe0b16615c08d09c3d80f5_2_1212x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/b/2/b265113068cd7b0589fe0b16615c08d09c3d80f5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1262×1040 66.1 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by billythekid** [comment]
<p>Hi folks!</p>
<p>Something went awry in the system here. Please don’t worry if you received one of these mistaken emails, this should be sorted now and we won’t be charging you for any incorrect amounts we might have emailed about, sorry for any alarm there!</p>
<p>You can find some more info on this here, the link that <a class="mention" href="/m1kep">@M1kep</a> posted above:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://www.githubstatus.com/incidents/vxvyrmy9w1vp">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/330170/181068182-d8801958-2425-4a26-88af-ca39cb3f13f1.png" class="site-icon" width="32" height="32">

      <a href="https://www.githubstatus.com/incidents/vxvyrmy9w1vp" target="_blank" rel="noopener">githubstatus.com</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/330170/181068183-63970b58-5ca7-4e4a-9d94-5e95b16325b0.png" class="thumbnail onebox-avatar" width="120" height="120">

<h3><a href="https://www.githubstatus.com/incidents/vxvyrmy9w1vp" target="_blank" rel="noopener">Incident with GitHub Actions and Codespaces</a></h3>

  <p>GitHub's Status Page - Incident with GitHub Actions and Codespaces.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>It should be back to normal if you refresh your billing page.</p>
<p>If you’re in any doubt or are concerned that any upcoming charges are incorrect do reach out to private support using our <a href="https://support.github.com/contact/account?subject=Incorrect+Charges&amp;type=billing">Account support form</a> where we might be able to go over your particular case in more detail with you.</p>
<p>Cheers,<br>
Billy</p>

**Comment by LeighS** [comment]
<p>This is still not resolved for us.</p>

**Comment by alexanderscott** [comment]
<p>same issue here.  cannot deploy or even download our github-hosted packages needed for developing.  pretty mission critical P1, and has been a few hours now.  communication from github has been mediocre to poor at best.</p>

**Comment by markvdputten** [comment]
<p>I just got an email that we are at 100% of our spending limit ($32,172.92 of $50).</p>
<p>This has to be some error…</p>

**Comment by steveklugherz** [comment]
<p>We are seeing the same (though the bill is $168M)</p>

**Comment by camoore1979** [comment]
<p>Yes. We are seeing the same in our <span class="mention">@polarislabs</span> org.</p>

**Comment by kruglov-dmitry** [comment]
<p>Same here, was little bit surprised when recieve similar notification that we a bit exceed our spending cap.</p>
<blockquote>
<p>deployments<br>
we partially mitigated it via self-hosted runners</p>
</blockquote>

**Comment by garrett-mesalabs** [comment]
<p>My organization is having the same issue, we got billed for roughly 4 million minutes in a repository that we have not run CI in recently.</p>

**Comment by jtlight** [comment]
<p>I just saw a spike to nearly 20 million minutes that is shutting down our actions.</p>
<p>Our reports showed two actions of over 9 million and 10 million minutes that that had the Username and Actions Workflow both blank.</p>
